index.html

```diff
@@ -528,6 +529,59 @@ <h2>Glossary</h2>
 </section>
 
 
+<section class="appendix" id="how-to-use">
+	<h3 id="howto">How to use this glossary</h3>
+	
+	<p>If you are writing a W3C specification, you can use the terminology in this document directly by importing this document as a cross-reference source. This may be especially helpful when using <a>character encoding</a> or <a>locale</a>-related terms found here. Such terms are common to the needs of many specifications and often refer to other definitions found in this glossary. In most cases, though, copying these definitions into your specification won't add value to your readers.</p>
+	
+    <p>This document is on the WG NOTE track in order to facilitate maintenance. This means that the definitions in the glossary are defined to be <em>informative</em> rather than <em>normative</em>.</p>
+    
+    <section id="infra-rel">
+		<h4>Relationship to the Infra Standard</h4>
+
+        <p>This document is meant to be used in conjunction with formal, normative definitions found in [[INFRA]]. When a term is defined in both documents, the [[INFRA]] version SHOULD be preferred. The definitions in both documents are maintained to be consistent with one-another.</p>
+        
+        <p>The I18N Glossary has many more terms specific to internationalization than are to be found in Infra, including many terms useful in defining or discussing the handling of text.</p>
```

Comment:
A key distinction between INFRA and our glossary that is missing here is that INFRA is HTML-specific.  Our glossary tries to provide definitions that are also usable outside the realm of HTML.

Reply:
Btw, should we say something about the relationship with the Unicode glossary?